packages/terra-button/src/Button.module.scss

```diff
@@ -78,6 +78,35 @@
       opacity: var(--terra-button-disabled-opacity, 0.3);
       pointer-events: none; // prevent pointer interaction for anchor elements
     }
+
+    &.is-selected.is-focused {
+      background-color: var(--terra-button-selected-and-focus-background-color, #52585b);
+      border-color: var(--terra-button-selected-and-focus-border-color, #52585b);
+      box-shadow: var(--terra-button-selected-and-focus-box-shadow, 0 0 1px 3px rgba(76, 178, 233, 0.5), 0 0 7px 4px rgba(76, 178, 233, 0.35));
+      color: var(--terra-button-selected-and-focus-color, #fff);
+    }
+
+    &.is-selected {
+      background-color: var(--terra-button-selected-background-color, #52585b);
+      border-color: var(--terra-button-selected-border-color, #52585b);
+      box-shadow: var(--terra-button-selected-box-shadow);
+      color: var(--terra-button-selected-color, #fff);
+    }
```

Comment:
I believe is-Selected is the only class we would need rest other styles should be same as default button ( which might be already added on _mixin.scss of button package.

Reply:
Updated along with renaming in https://github.com/cerner/terra-core/pull/3940/commits/cfcc95056c0614cdaaafcf289f610483fa562d38